src/categories/categories.controller.ts

```diff
@@ -1,34 +1,41 @@
 import { Controller, Get, Post, Body, Patch, Param, Delete } from '@nestjs/common';
+import { ApiOperation } from '@nestjs/swagger';
+import { Ok, ZaLaResponse } from '../common/helpers/response';
 import { CategoriesService } from './categories.service';
 import { CreateCategoryDto } from './dto/create-category.dto';
 import { UpdateCategoryDto } from './dto/update-category.dto';
+import { Category } from '../entities/category.entity';
 
```

Comment:
i can see the swagger API tag

Reply:
@TijanAyo update your branch with the latest changes from the main branch

Reply:
Omo I don't know what is wrong with my branch... i pulled before merging